mongodbatlas/mongodbatlas.go

```diff
@@ -564,3 +566,38 @@ func setListOptions(s string, opt interface{}) (string, error) {
 	origURL.RawQuery = origValues.Encode()
 	return origURL.String(), nil
 }
+
+// ServiceVersion represents version information.
+type ServiceVersion struct {
+	GitHash string
+	Version string
+}
+
+// String serializes VersionInfo into string.
+func (v *ServiceVersion) String() string {
+	return fmt.Sprintf("gitHash=%s; versionString=%s", v.GitHash, v.Version)
+}
+
+func parseVersionInfo(s string) *ServiceVersion {
+	if s == "" {
+		return nil
+	}
+
+	var result ServiceVersion
+	pairs := strings.Split(s, ";")
+	for _, pair := range pairs {
+		keyvalue := strings.Split(strings.TrimSpace(pair), "=")
+		switch keyvalue[0] {
+		case "gitHash":
+			result.GitHash = keyvalue[1]
+		case "versionString":
+			result.Version = keyvalue[1]
+		}
+	}
+	return &result
+}
+
+// GetServiceVersion parses version information returned in the response.
+func (resp *Response) GetServiceVersion() *ServiceVersion {
```

Comment:
[nit] getters are non idiomatic on go
```suggestion
func (resp *Response) ServiceVersion() *ServiceVersion {
```

Reply:
I know it does not sound right but I did not think `Get` as a getter here 😄